### PR TITLE
feat(pluginutils): normalizePath

### PR DESCRIPTION
**Pull request by eight04**
## Rollup Plugin Name: `pluginutils`

This PR contains:

- [x] bugfix
- [x] feature
- [ ] refactor
- [ ] documentation
- [ ] other

Are tests included?

- [x] yes (_bugfixes and features will not be merged without tests_)
- [ ] no

Breaking Changes?

- [ ] yes (_breaking changes will not be merged unless absolutely necessary_)
- [x] no

If yes, then include "BREAKING CHANGES:" in the first commit message body, followed by a description of what is breaking. 

List any relevant issue numbers:

### Description

This PR includes:

1. A new utility `normalizePath`.
2. Fixes a bug on Windows. On Windows 10, test failed because `path.resolve` creates patterns including backward slash `\`. This patch adds a normalization similar to `createFilter`:
https://github.com/rollup/plugins/blob/55a9cb20d7686750aa104d6a45a7d6834f57ba4a/packages/pluginutils/src/createFilter.ts#L15-L17
3. Small quotes typo in README.

### CODE REVIEW

**Comment by shellscape** [comment]
@eight04 please take a look at the last comment

**Comment by eight04** [comment]
How does typescript plugin use the normalization? If it also relates to picomatch then we can add a `compilePicomatch` function.

Or, do you want a small util like this:

```js
export function normalizePathSep(path) {
  return path
    .split(sep)
    .join('/');
}
```

**Comment by NotWoods** [comment]
Just a small util like that, although I prefer the [implementation in the typescript plugin](https://github.com/rollup/plugins/blob/master/packages/typescript/src/index.ts#L29) as it uses constants from node.

File: packages/pluginutils/README.md [hunk 1]
```diff
@@ -88,9 +88,9 @@ export default function myPlugin(options = {}) {
         },
         leave(node) {
           if (node.scope) scope = scope.parent;
-        }
+        },
```
Comment (eight04) [inline packages/pluginutils/README.md:91]: I think these were added by prettier.

**Review by NotWoods — APPROVED** [review]

**Review by pnevares — APPROVED** [review]

**Comment by shellscape** [comment]
thanks!

**Pull request merged**